iwyu.cc

```diff
@@ -3623,7 +3651,7 @@ class IwyuAstConsumer
   // Called whenever a variable, function, enum, etc is used.
   bool VisitDeclRefExpr(clang::DeclRefExpr* expr) {
     if (CanIgnoreCurrentASTNode())  return true;
-    ReportDeclUse(CurrentLoc(), expr->getDecl());
+    ReportDeclUse(CurrentLoc(), expr->getFoundDecl());
```

Comment:
Ah, makes sense! Yeah, let's change `VisitDeclRefExpr` to be backward-compatible, and we can think about simplifying later (that `UsingShadowDecl` cast seems like it's happening everywhere now :-))